package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ionic-plugin-deeplinks",
-  "version": "1.0.21",
+  "name": "@spoonconsulting/ionic-plugin-deeplinks",
+  "version": "1.0.0",
```

Comment:
@dinitri-ragoo I would rather keep upgrading the version number. 1.0.22 ? What do you think ?